app/views/survey_responses/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Index</h1>
+<table>
+  <tr>
+    <th>Survey Date</th>
+    <th>Confidence Level</th>
+    <th>Difficulty of Material</th>
+    <th>Notes</th>
+  </tr>
```

Comment:
I imagine the design side of things is coming later?

Reply:
Yup. Maybe I’ll use elm 😂